src/Crate/PDO/PDOStatement.php

```diff
@@ -202,7 +202,10 @@ public function execute($input_parameters = null)
             if (is_int($parameter) && $zero_based) {
                 $parameter++;
             }
-            $this->bindValue($parameter, $value);
+            // to correctly save arrays and objects, we need to pass
```

Comment:
Not sure why you are automating this sort of operation. You are supposed to pass in `PDO::PARAM_ARRAY` or `PDO::PARAM_OBJECT` (manually) when binding parameters.